samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+       "|000010⟩\t0 + 0𝑖\n",
+       "|000011⟩\t0 + 0𝑖\n",
+       "|000100⟩\t0 + 0𝑖\n",
+       "|000101⟩\t0 + 0𝑖\n",
+       "|000110⟩\t0 + 0𝑖\n",
+       "|000111⟩\t0.5000000000000001 + 0𝑖\n",
+       "|001000⟩\t0 + 0𝑖\n",
+       "|001001⟩\t0 + 0𝑖\n",
+       "|001010⟩\t0 + 0𝑖\n",
+       "|001011⟩\t0 + 0𝑖\n",
+       "|001100⟩\t0 + 0𝑖\n",
+       "|001101⟩\t0 + 0𝑖\n",
+       "|001110⟩\t0 + 0𝑖\n",
+       "|001111⟩\t0 + 0𝑖\n",
+       "|010000⟩\t0 + 0𝑖\n",
+       "|010001⟩\t0 + 0𝑖\n",
+       "|010010⟩\t0 + 0𝑖\n",
+       "|010011⟩\t0 + 0𝑖\n",
+       "|010100⟩\t0 + 0𝑖\n",
+       "|010101⟩\t0 + 0𝑖\n",
+       "|010110⟩\t0 + 0𝑖\n",
+       "|010111⟩\t0 + 0𝑖\n",
+       "|011000⟩\t0 + 0𝑖\n",
+       "|011001⟩\t0 + 0𝑖\n",
+       "|011010⟩\t0 + 0𝑖\n",
+       "|011011⟩\t0 + 0𝑖\n",
+       "|011100⟩\t0 + 0𝑖\n",
+       "|011101⟩\t0 + 0𝑖\n",
+       "|011110⟩\t0 + 0𝑖\n",
+       "|011111⟩\t0 + 0𝑖\n",
+       "|100000⟩\t0 + 0𝑖\n",
+       "|100001⟩\t0 + 0𝑖\n",
+       "|100010⟩\t0 + 0𝑖\n",
+       "|100011⟩\t0 + 0𝑖\n",
+       "|100100⟩\t0 + 0𝑖\n",
+       "|100101⟩\t0 + 0𝑖\n",
+       "|100110⟩\t0 + 0𝑖\n",
+       "|100111⟩\t0 + 0𝑖\n",
+       "|101000⟩\t0 + 0𝑖\n",
+       "|101001⟩\t0 + 0𝑖\n",
+       "|101010⟩\t0 + 0𝑖\n",
+       "|101011⟩\t0 + 0𝑖\n",
+       "|101100⟩\t0 + 0𝑖\n",
+       "|101101⟩\t0 + 0𝑖\n",
+       "|101110⟩\t0 + 0𝑖\n",
+       "|101111⟩\t0 + 0𝑖\n",
+       "|110000⟩\t0 + 0𝑖\n",
+       "|110001⟩\t0 + 0𝑖\n",
+       "|110010⟩\t0 + 0𝑖\n",
+       "|110011⟩\t0 + 0𝑖\n",
+       "|110100⟩\t0 + 0𝑖\n",
+       "|110101⟩\t0 + 0𝑖\n",
+       "|110110⟩\t0 + 0𝑖\n",
+       "|110111⟩\t0 + 0𝑖\n",
+       "|111000⟩\t0.5000000000000001 + 0𝑖\n",
+       "|111001⟩\t0 + 0𝑖\n",
+       "|111010⟩\t0 + 0𝑖\n",
+       "|111011⟩\t0 + 0𝑖\n",
+       "|111100⟩\t0 + 0𝑖\n",
+       "|111101⟩\t0 + 0𝑖\n",
+       "|111110⟩\t0 + 0𝑖\n",
+       "|111111⟩\t0.5000000000000001 + 0𝑖"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Collapse and show final state:\n",
+      "[Zero,Zero,Zero,One,One,One]\n",
+      "Cleaning up...\n",
+      "Done!\n"
+     ]
+    },
+    {
+     "data": {
+      "application/x-qsharp-data": "{\"@type\":\"tuple\"}",
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%simulate PrepareAndMeasureGhzState nQubits=6 nClusters=2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dying-armstrong",
+   "metadata": {},
+   "source": [
+    "In the cell above increase the `nQubits` parameter (by multiples of `nClusters` parameter) such that the simulation becomes evidently slow (e.g. `nQubits`: 8, 10, 12, 14, ...; `nClusters`: 2). On some laptops the simulation becomes slow at `nQubits=20`. Don't make large steps otherwise the simulation can run too long.\n",
```

Comment:
```suggestion
    "In the cell above increase the `nQubits` input (by multiples of `nClusters` parameter) such that the simulation becomes slow (e.g. `nQubits`: 8, 10, 12, 14, ...; `nClusters`: 2). On some laptops the simulation becomes slow at `nQubits=20`. Don't make large steps otherwise the simulation can run too long.\n",
```

Other notes:
- If this is intended for use in the portal gallery as well, "laptop" doesn't have a lot of meaning here.